src/components/register/steps/ticket/type.tsx

```diff
@@ -20,9 +21,15 @@ const TicketTypeGrid = styled.div`
 `
 
 const TicketType = (_: RouteComponentProps) => {
-	const { register, handleSubmit } = useForm<{ type: TicketTypeModel['type'] }>()
+	const { register, watch, handleSubmit } = useForm<{ type: TicketTypeModel['type'] }>()
 	const dispatch = useAppDispatch()
 
+	useEffect(() => {
+		const subscription = watch(data => dispatch(ChangeTicketType.create(data.type)))
+
+		return () => subscription.unsubscribe()
```

Comment:
Since we need to return a function, can we not just return `subscription.unsubscribe`?